edx_event_bus_kafka/consumer/event_consumer.py

```diff
@@ -4,8 +4,14 @@
 
 import logging
 
-from confluent_kafka import DeserializingConsumer, KafkaError
-from confluent_kafka.schema_registry.avro import AvroDeserializer
+try:
+    from confluent_kafka import DeserializingConsumer, KafkaError
+    from confluent_kafka.schema_registry.avro import AvroDeserializer
+except:
+    DeserializingConsumer = None
```

Comment:
Consider:
```
confluent_kafka = None
```
Would the code read better if everything just checked:
```
if confluent_kafka:
```
That's what we do for New Relic.
```

Reply:
Good call. Will go with that.

Reply:
@robrap do you know of any ADRs or OEPs detailing how the decision was made to use this pattern for New Relic? Github code search is not forthcoming with any, but it'd be a useful reference if it exists.

Reply:
No ADR. I think the code just reads cleanly. See example: https://github.com/openedx/edx-django-utils/blob/master/edx_django_utils/monitoring/internal/middleware.py#L160-L161. I think it would be reasonable to note in this ADR the pattern of code that we'd follow.